packages/contracts/package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "test": "hardhat test",
     "build": "hardhat compile",
-    "build:npm": "rollup --config rollup.config.ts",
+    "build:npm": "rollup --config rollup.config.ts && cp -a ./src/* ./dist",
```

Comment:
The resulting import path would be 

import "@aragon/core-contracts/**dist**/core/dao/DAO.sol";

I would do it like OZ, ENS, and GSN and directly put it in the top-level folder.

import "@aragon/core-contracts/core/dao/DAO.sol";

Moreover, I would only be more selective with the folders:

Include
- `core` 
- `framework`
- `utils`

Potentially include
- `plugins`
- `token`

Exclude
- `test`

Reply:
The problem with this is that this forces us to copy the source folders to the root of the `contract` folder, which is very cumbersome

Reply:
We could also whitelist `dist` and `src` and then imports would look like this
Alternatively, having a kind of "chroot" within `dist`.  Copying the package.json file there and doing the publish from there

Reply:
I agree. `dist` shouldn't be in the path and for the first time, let's include plugins and token. And ofc exclude `test`.